Cargo.toml

```diff
@@ -39,7 +39,7 @@ cpp_demangle = { default-features = false, version = "0.4.0", optional = true, f
 ] }
 
 [target.'cfg(any(windows, target_os = "cygwin"))'.dependencies]
-windows-targets = "0.52.6"
+windows-targets = ">=0.52.6,<0.54.0"
```

Comment:
Driveby note; this is a pretty unusual version spec, it should come with a comment explaining what's going on

Reply:
Added a comment in the latest rebased commit.

TBH, I'm not sure what commentary to add besides "this is what the version spec. does". Did you have more in mind?